src/lib/codeSamples/buildRequest.ts

```
@@ -25,31 +25,41 @@
-function getHeaders(variables: Record<string, string>, headerParameters: OpenAPIV3.ParameterObject[], authScheme: PlaygroundSecurityScheme | null) {
+function getHeaders(variables: Record<string, string>, headerParameters: OpenAPIV3.ParameterObject[], authorizations: PlaygroundSecurityScheme | PlaygroundSecurityScheme[] | null) {
   const headers = new Headers({})
 
   processParameters(variables, headerParameters, (key: string, value: string) => {
     headers.set(key, value)
   })
 
-  if (!authScheme) {
-    return headers
+  if (authorizations && !Array.isArray(authorizations)) {
+    authorizations = [authorizations]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)